modules/viewing-current-log-level.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cli/debugging-issues.adoc
+:_content-type: PROCEDURE
+[id="viewing-current-log-level_{context}"]
+= Viewing current log level
+
+You can change the log level to see more or less information in Central logs.
+
+.Procedure
+
+* Run the following command to view the current log level:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl "$ROX_CENTRAL_ADDRESS" central debug log
```

Comment:
The correct variable is `ROX_ENDPOINT` and we do not need to set this explicitly here.

Since we initially documented it together with the `ROX_API_TOKEN` we can drop this here.

If we want to keep it, let's use it like:
```bash
export ROX_ENDPOINT=<host:port>
roxctl image check --image=<image name>
```